pkg/ingester/ingester.go

```diff
@@ -48,17 +48,17 @@ type Config struct {
 	MaxTransferRetries int `yaml:"max_transfer_retries,omitempty"`
 
 	// Config for chunk flushing.
-	FlushCheckPeriod  time.Duration
-	RetainPeriod      time.Duration
-	MaxChunkIdle      time.Duration
-	MaxStaleChunkIdle time.Duration
-	FlushOpTimeout    time.Duration
-	MaxChunkAge       time.Duration
-	ChunkAgeJitter    time.Duration
-	ConcurrentFlushes int
-	SpreadFlushes     bool
-
-	RateUpdatePeriod time.Duration
+	FlushCheckPeriod  time.Duration `yaml:"flush_period"`
+	RetainPeriod      time.Duration `yaml:"retain_period"`
+	MaxChunkIdle      time.Duration `yaml:"max_chunk_idle"`
+	MaxStaleChunkIdle time.Duration `yaml:"max_stale_chunk_idle"`
```

Comment:
```suggestion
	MaxChunkIdle      time.Duration `yaml:"max_chunk_idle_time"`
	MaxStaleChunkIdle time.Duration `yaml:"max_stale_chunk_idle_time"`
```

Reply:
@pstibrany would you suggest we suffix all duration values like this with `_time`. Similar to how counters in Prometheus are suffixed with `_total`. I think that might be a little clunky but does provide value.